src/Yoti.Auth/DocScan/DocScanService.cs

```diff
@@ -0,0 +1,211 @@
+                    Response.CreateYotiExceptionFromStatusCode<DocScanException>(response);
+                }
+            }
+        }
+
+        public async Task<MediaValue> GetMediaContent(string sdkId, AsymmetricCipherKeyPair keyPair, string sessionId, string mediaId)
+        {
+            Validation.NotNullOrEmpty(sdkId, nameof(sdkId));
+            Validation.NotNull(keyPair, nameof(keyPair));
+            Validation.NotNull(sessionId, nameof(sessionId));
+            Validation.NotNull(mediaId, nameof(mediaId));
+
+            string mediaContentPath = MediaEndpoint(sessionId, mediaId);
+            _logger.Info($"Fetching media from '{mediaContentPath}'");
+
+            Request getMediaContentRequest = GetSignedRequestBuilder()
+                .WithKeyPair(keyPair)
+                .WithHttpMethod(HttpMethod.Get)
+                .WithBaseUri(ApiUri)
+                .WithEndpoint(mediaContentPath)
+                .WithQueryParam("sdkId", sdkId)
+                .Build();
+
+            using (HttpResponseMessage response = await getMediaContentRequest.Execute(_httpClient).ConfigureAwait(false))
+            {
+                if (!response.IsSuccessStatusCode)
+                {
+                    Response.CreateYotiExceptionFromStatusCode<DocScanException>(response);
+                }
+
+                string contentType = response.Content.Headers.ContentType.MediaType;
+
+                return new MediaValue(contentType, response.Content.ReadAsByteArrayAsync().Result);
+            }
+        }
+
+        public void DeleteMediaContent(string sdkId, AsymmetricCipherKeyPair keyPair, string sessionId, string mediaId)
+        {
+            Validation.NotNullOrEmpty(sdkId, nameof(sdkId));
+            Validation.NotNull(keyPair, nameof(keyPair));
+            Validation.NotNull(sessionId, nameof(sessionId));
+            Validation.NotNull(mediaId, nameof(mediaId));
+
+            string mediaContentPath = MediaEndpoint(sessionId, mediaId);
+            _logger.Info($"Deleting media at '{mediaContentPath}'");
+
+            Request deleteMediaContentRequest = GetSignedRequestBuilder()
+                .WithKeyPair(keyPair)
+                .WithHttpMethod(HttpMethod.Delete)
+                .WithBaseUri(ApiUri)
+                .WithEndpoint(mediaContentPath)
+                .WithQueryParam("sdkId", sdkId)
+                .Build();
+
+            using (HttpResponseMessage response = deleteMediaContentRequest.Execute(_httpClient).Result)
+            {
+                if (!response.IsSuccessStatusCode)
+                {
+                    Response.CreateYotiExceptionFromStatusCode<DocScanException>(response);
+                }
+            }
+        }
+
+        private static Uri GetApiUri()
+        {
+            if (!string.IsNullOrEmpty(Environment.GetEnvironmentVariable("YOTI_API_URL")))
+            {
+                return new Uri(Environment.GetEnvironmentVariable("YOTI_API_URL"));
+            }
```

Comment:
```suggestion
            if (!string.IsNullOrEmpty(Environment.GetEnvironmentVariable("YOTI_DOC_SCAN_API_URL")))
            {
                return new Uri(Environment.GetEnvironmentVariable("YOTI_DOC_SCAN_API_URL"));
            }
```

Reply:
Changed in https://github.com/getyoti/yoti-dotnet-sdk/pull/196/commits/ac7d496eda50a7a0d17d9255f7588076b4225cf3